mixins/test/label-mixin.test.js

```diff
@@ -0,0 +1,162 @@
+
+import { defineCE, expect, fixture, html, nextFrame } from '@open-wc/testing';
+import { LabelledMixin, LabelMixin } from '../labelled-mixin.js';
+import { LitElement } from 'lit-element/lit-element.js';
+
+const labelledTag = defineCE(
+	class extends LabelledMixin(LitElement) {
+		render() {
+			return html`
+				<input type="text" aria-label="${this.label}">
+			`;
+		}
+	}
+);
+
+const labelTag = defineCE(
+	class extends LabelMixin(LitElement) {
+		static get properties() {
+			return {
+				text: { type: String }
+			};
+		}
+		render() {
+			return html`
+				<span>${this.text}</span>
+			`;
+		}
+		updated(changedProperties) {
+			super.updated(changedProperties);
+			this.updateLabel(this.text);
+		}
+	}
+);
+
+describe.only('LabelMixin', () => {
+
+	let elem;
+
+	beforeEach(async() => {
+		elem = await fixture(`
+			<${labelTag} text="the label value"></${labelTag}>
+		`);
+	});
+
+	it('reflects label value', async() => {
+		expect(elem.getAttribute('_label')).to.equal('the label value');
+	});
+
+	it('reflects label value change', async() => {
+		elem.text = 'new label value';
+		await nextFrame();
+		expect(elem.getAttribute('_label')).to.equal('new label value');
+	});
+
+});
+
+describe.only('LabelledMixin', () => {
```

Comment:
Finally me catching this and not causing it! 🥇 

```suggestion
describe('LabelledMixin', () => {
```